src/api/app/views/webui/shared/_download_repository_link.html.haml

```diff
@@ -1,5 +1,11 @@
 - url = "#{download_url}/#{project.to_s.gsub(/:/, ':/')}/#{repository}"
-%li.list-inline-item
-  = link_to(url, title: 'Go to download repository', class: 'nav-link') do
-    %i.fas.fa-download
-    Go to download repository
+- result = Buildresult.summary(project)
+- published = result[repository.name].select {|s| s.state != "unpublished" }
```

Comment:
## Flags
- A `Flag` is an on/off switch for *something*. Like
  - allow (`status=enable`) publishing (`flag=publish`) for the `Architecture` x86_64  (`architecture_id=1`)
  - do not (`status=disable`) build (`flag=build`) for a specific `Repository`  (`repo=openSUSE_Tumbleweed`)
  - allow (`status=enable`) building  (`flag=build`) for a specific `Repository`  (`repo=openSUSE_Tumbleweed`) in the `Architecture`  x86_64  (`architecture_id=1`)
- More specific `Flags` overwrite unspecific flags. In the example above the third `Flag` overwrites the second.
- `Flags` have global defaults ([FlagHelper::TYPES](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/helpers/flag_helper.rb#L6))

## Project
- A Project **can** have many `Flags`
- If a `Project` does not have a `Flag` for a specific thing, the global default is taken.

## Package
- A `Package` **can** have many `Flag`s
- If a `Package` does not have a `Flag` for a specific thing, the `Flag`s of its `Project` are taken into account.

Reply:
Okay, I've changed the code to use `Flag`s